### PR TITLE
Image refresh for fedora-40

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for fedora-40
 * [x] image-refresh fedora-40

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh fedora-40 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-40-20240427-225328

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-40-ce05650c-20240427-223720/log.html

**Comment by jelly** [comment]
Everything abrt related broke, needs work.

**Comment by jelly** [comment]
```
Changed:
  bind-libs (32:9.18.24-1.fc40 -> 32:9.18.26-1.fc40)
  bind-license (32:9.18.24-1.fc40 -> 32:9.18.26-1.fc40)
  bind-utils (32:9.18.24-1.fc40 -> 32:9.18.26-1.fc40)
  cockpit (314-1.fc40 -> 315-1.fc40)
  cockpit-bridge (314-1.fc40 -> 315-1.fc40)
  cockpit-system (314-1.fc40 -> 315-1.fc40)
  cockpit-ws (314-1.fc40 -> 315-1.fc40)
  forge-srpm-macros (0.2.0-3.fc40 -> 0.3.1-1.fc40)
  fwupd-efi (1.4-1.fc38 -> 1.6-1.fc40)
  grub2-common (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-efi-ia32 (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-efi-x64 (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-efi-x64-modules (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-pc (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-pc-modules (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-tools (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-tools-efi (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-tools-extra (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  grub2-tools-minimal (1:2.06-119.fc40 -> 1:2.06-121.fc40)
  libX11 (1.8.7-3.fc40 -> 1.8.9-1.fc40)
  libX11-common (1.8.7-3.fc40 -> 1.8.9-1.fc40)
  libX11-xcb (1.8.7-3.fc40 -> 1.8.9-1.fc40)
  libgusb (0.4.8-3.fc40 -> 0.4.9-1.fc40)
  libipa_hbac (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  libproxy (0.5.3-5.fc40 -> 0.5.5-1.fc40)
  libsss_autofs (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  libsss_certmap (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  libsss_idmap (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  libsss_nss_idmap (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  libsss_sudo (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  libxmlb (0.3.18-1.fc40 -> 0.3.19-1.fc40)
  linux-system-roles (1.76.0-1.fc40 -> 1.76.2-1.fc40)
  mock-core-configs (40.2-1.fc40 -> 40.3-1.fc40)
  nbdkit (1.37.13-1.fc40 -> 1.38.0-1.fc40)
  nbdkit-basic-filters (1.37.13-1.fc40 -> 1.38.0-1.fc40)
  nbdkit-basic-plugins (1.37.13-1.fc40 -> 1.38.0-1.fc40)
  nbdkit-curl-plugin (1.37.13-1.fc40 -> 1.38.0-1.fc40)
  nbdkit-selinux (1.37.13-1.fc40 -> 1.38.0-1.fc40)
  nbdkit-server (1.37.13-1.fc40 -> 1.38.0-1.fc40)
  nbdkit-ssh-plugin (1.37.13-1.fc40 -> 1.38.0-1.fc40)
  pciutils-libs (3.11.1-1.fc40 -> 3.12.0-1.fc40)
  pcp (6.2.0-1.fc40 -> 6.2.1-1.fc40)
  pcp-conf (6.2.0-1.fc40 -> 6.2.1-1.fc40)
  pcp-libs (6.2.0-1.fc40 -> 6.2.1-1.fc40)
  pcp-selinux (6.2.0-1.fc40 -> 6.2.1-1.fc40)
  python3-gobject-base (3.48.1-1.fc40 -> 3.48.2-1.fc40)
  python3-libipa_hbac (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  python3-sss (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  python3-sss-murmur (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  python3-sssdconfig (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  qt6-srpm-macros (6.6.2-1.fc40 -> 6.7.0-1.fc40)
  sssd (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-ad (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-client (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-common (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-common-pac (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-dbus (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-idp (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-ipa (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-kcm (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-krb5 (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-krb5-common (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-ldap (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-nfs-idmap (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-passkey (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-proxy (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  sssd-tools (2.9.4-4.fc40 -> 2.9.4-7.fc40)
  xen-libs (4.18.1-1.fc40 -> 4.18.2-1.fc40)
  xen-licenses (4.18.1-1.fc40 -> 4.18.2-1.fc40)
```

**Comment by jelly** [comment]
Needs a rebase for CentOS image and dropped test.

**Comment by jelly** [comment]
It's always SELinux:

```
SELinux is preventing abrt-dump-journ from write access on the sock_file io.systemd.Machine. For complete SELinux messages run: sealert -l 25180c91-47f3-4720-8af5-b726a9562773
```

File: naughty/fedora-40/6621-selinux-abrt-dynamicuser [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*avc:  denied  { write } for * comm="abrt-dump-journ" name="io.systemd.*" dev="tmpfs" * scontext=system_u:system_r:abrt_dump_oops_t:s0 tcontext=system_u:object_r:systemd_userdbd_runtime_t:s0 tclass=sock_file*
```
Comment (martinpitt) [inline naughty/fedora-40/6621-selinux-abrt-dynamicuser:1]: Typo -- this is 6**2**21, not 6**6**21

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Comment by martinpitt** [comment]
@jelly When you force-push, for my sake it's enough to run fedora-40/other@cockpit-project/cockpit, I know that this was green.

File: naughty/fedora-40/6621-selinux-abrt-dynamicuser (hunk 1 above)
Comment (jelly) [reply naughty/fedora-40/6621-selinux-abrt-dynamicuser:1]: Ah the copy pasta https://github.com/cockpit-project/bots/commit/c5b6dfdc9b204b5dad5239f01d4ed0af9fe9f4ec :)

Comment (jelly) [reply naughty/fedora-40/6621-selinux-abrt-dynamicuser]: Also adjusted 41.

Comment (martinpitt) [reply naughty/fedora-40/6621-selinux-abrt-dynamicuser]: *brown paperpag*, thanks!

**Review by martinpitt — APPROVED** [review]
Cheers!

**Comment by martinpitt** [comment]
[known flake](https://cockpit-logs.us-east-1.linodeobjects.com/pull-6308-85085062-20240430-091157-fedora-40-other-cockpit-project-cockpit/log.html#30), it's on the pilot board, ignoring.

**Pull request merged**